test_python/hello2.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3
+name, age, location = input("Enter your name, age and location with a space in between:").split()
```

Comment:
```suggestion
name, age, location = input("Enter your name, age and location with a space in between:").split()
```
`name = input("Enter your name")` `age = input("Enter your age")` and so on.